src/runtime-rs/crates/resource/src/cgroups/utils.rs

```diff
@@ -4,6 +4,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+use std::path::Path;
+
+// prepend a kata specific string to oci cgroup path to
+// form a different cgroup path, thus cAdvisor couldn't
+// find kata containers cgroup path on host to prevent it
+// from grabbing the stats data.
+const CGROUP_KATA_PREFIX: &str = "kata";
+
+// DEFAULT_RESOURCE_CONTROLLER_ID runtime-determined location in the cgroups hierarchy.
+const DEFAULT_RESOURCE_CONTROLLER_ID: &str = "vc";
```

Comment:
I have two questions here:

1. What does `vc` stand for? VirtContainer?
2. Bear with me in the case I'm wrong. The `DEFAULT_RESOURCE_CONTROLLER_ID` is to avoid that the path is empty or "/". But why is the `kata` prefix not needed?

Reply:
1. The `vc` stands for VirtContainer.
2. Sorry, I missed it, I added it.